compat/src/main/scala-2.11_2.12/scala/collection/compat/PackageShared.scala

```diff
@@ -55,6 +55,15 @@ private[compat] trait PackageShared {
     simpleCBF(builder)
   }
 
+  implicit def genericOrderedCompanionToCBF[A, CC[X] <: Traversable[X]](
+      fact: GenericOrderedCompanion[CC])(implicit ordering: Ordering[A]): CanBuildFrom[Any, A, CC[A]] = {
+    val builder: m.Builder[A, CC[A]] = fact match {
+      case m.PriorityQueue => new IdentityPreservingBuilder[A, m.PriorityQueue](m.PriorityQueue.newBuilder[A])
+      case _ => fact.newBuilder[A]
+    }
+    simpleCBF(builder)
+  }
```

Comment:
I think we should just call `simpleCBF(fact.newBuilder[A])`. The `IdentityPreservingBuilder` is not necessary here because `to[m.PriorityQueue` was not identity-preserving in 2.12.